### PR TITLE
Thu-Mai's Turtle Exercise

### PR DESCRIPTION
**Pull request by tlchristian**
I have completed the Turtle Exercise after editing my post markup.  After removing what I was hoping was the proper syntax to escape the \ character, the post appeared on my site correctly.  Please see the results here: http://tlchristian.github.io/spring2014/2014/01/16/thumaiturtle.html.

### CODE REVIEW

**Comment by gerbal** [comment]
@tlchristian Sometimes github takes a while to update the cache on all of their servers. If you page isn't showing up, check you email for build-errors and leave a link to http://tlchristian.github.io/spring2014 in the description. You don't have to link the the specific post, just your github.io site.

File: _posts/2014-01-15-thumaiturtle.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+title: "Thu-Mai's Turtle Exercise"
+---
+**Does Turtle Love You or Love You Not?**
+
+For this Turtle Exercise, I ~~decided to make~~ ended up making an interactive love prediction flower. I wanted to play with user inputs, polygons, colors, and a little bit of cute. As I kept playing with the flower graphic, I decided to add some if-else code and write text in the window. This is what makes up the "prediction" part.
+
+When you run the code, you will be asked to enter in a number of flower petals. The turtle will draw a flower with that number of randomly colored petals. Based on the number of petals selected, the turtle will tell you whether he loves you or loves you not.\*
+
+**The code:**
+```
```
Comment (gerbal) [inline _posts/2014-01-15-thumaiturtle.md]: your code block won't build correctly unless it is separated by a new line from the text above it.

**Comment by tlchristian** [comment]
Ah ha!  The problem I found was in my markup. I wanted to escape the \* key by using .  I guess it doesn't like that!  Thanks for your help, Grant!

**Comment by tlchristian** [comment]
I actually removed more text formatting.  I wish there were more options (or I wish I can find correct ways) for formatting text.

**Comment by tlchristian** [comment]
Sorry, didn't mean to close the request.  Request reopened.

**Comment by tlchristian** [comment]
Just added minor edits to text.  I won't touch it anymore, I promise.

**Comment by eah13** [comment]
@tlchristian nice work explaining what was going on with your commits.  I was able to quickly and easily come up to speed on everything you'd done, even though you ran into a series of errors and made several changes.

And, on top of that, great code!  I love using the input the change the way the image is displayed.

**Pull request merged**